spec/index.html

```diff
@@ -1466,164 +1473,233 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
+      None of these situations can occur if the input graph was produced by the [=classicize=] transformation.
+    </p>
 
-    <p class=note>
-      In <a href="#qtt-fresh-bnode">step 4.3</a> of this algorithm,
-      it is assumed that the blank node <var>b</var> is distinct from any blank node already in use in any dataset at hand,
-      in particular the dataset being processed in the invoking <a href="#section-unstar-algo">`unstar`</a> algorithm.
-      Some implementations may require that `unstar` passes that dataset to `quote-triple-term` to ensure this constraint.
+    <p>To revert a [=classicized=] [=RDF dataset=] to its original form, the transformation above is applied to its [=default graph=] and to each of its [=named graphs=].
     </p>
 
+    <p>Note that this transformation has no effect on any [=RDF graph=] or [=RDF dataset=] that does not use the `rdf:TripleTerm` type,
+      including [=Full=] graphs or datasets containing [=triple terms=].
+      This makes this transformation <em>idempotent</em> as intended.
+    </p>
   </section>
 
-  <section id="section-unstar-caveat">
-    <h2>Limitations of the `unstar` algorithm</h2>
+  <section id="section-classicize-caveat">
+    <h2>Limitations</h2>
 
-    <p>Steps <a href="#unstar-error1">4.1.1</a> and <a href="#unstar-error2">4.2.1</a> of the <a href="#section-unstar-algo">`unstar`</a> algorithm exit with an error.
-      This will occur if the input dataset contains at the same time <a>triple terms</a> and a graph named `rdf:unstarMetadata`.
-      The `unstar` algorithm is therefore not strictly universal as it can not transform this particular kind of datasets.
+    <p>The two transformations above are explicitly not supporting graphs or datasets containing at the same time a [=triple term=]
+      and an [=asserted triple=] (<var>b</var>, `rdf:type`, `rdf:TripleTerm`) where <var>b</var> is a [=blank node=].
+      This means, in particular, that the [=classicize=] transformation is not strictly <em>universal</em>.
     </p>
 
     <p>This limitation should not be an issue in practice.
-      The special graph name `rdf:unstarMapping` is unlikely to be in used in any published dataset,
-      as it was not defined in the RDF namespace prior to this specification.
+      The `rdf:TripleTerm` type is unlikely to be in used in any published graph or dataset,
+      as it was not defined prior to this specification.
       For this reason, using it would actually have been bad practice.
-      As for future datasets, their authors should consider the graph name `rdf:unstarMetadata` to be reserved, in order to prevent interference with the `unstar` algorithm.
+      As for future graphs and datasets, their authors should consider this type to be reserved, in order to prevent interference with the [=classicize=] transformation.
```

Comment:
```suggestion
      For future graphs and datasets, this type should be considered to be reserved for use within the [=classicize=] transformation, and not used otherwise.
```